src/components/volunteer/VolunteerStepsMobile.jsx

```diff
@@ -0,0 +1,302 @@
+import StepContent from '@mui/material/StepContent';
+import StepLabel from '@mui/material/StepLabel';
+import Stepper from '@mui/material/Stepper';
+import Typography from '@mui/material/Typography';
+
+const steps = [
+  {
+    label: 'Self-Onboarding & Initial Contact',
+    description: (
+      <>
+        <Typography variant="h6">Get familiar with CODE PDX</Typography>
+        <ol>
+          <li>
+            Read our{' '}
+            <a
+              href="https://github.com/codeforpdx/codeofconduct"
+              target="_blank"
+              rel="noreferrer"
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              Code of Conduct
+            </a>
+          </li>
+          <li>
+            Complete our{' '}
+            <a
+              href="https://docs.google.com/forms/d/e/1FAIpQLSfu4u9FB00RXd2TncbMLws_Ckot-vYqPK0oUHr81uLDjThE7Q/viewform"
+              target="_blank"
+              rel="noreferrer"
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              onboarding form
+            </a>
+          </li>
+          <li>
+            Review our{' '}
+            <a
+              href="https://github.com/codeforpdx"
+              target="_blank"
+              rel="noreferrer"
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              current projects
+            </a>
+          </li>
+          <li>
+            Read the{' '}
+            <a
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              ADA Compliance Guide
+            </a>
+            , all CODE PDX projects are inclusive by design
+          </li>
+          <li>
+            Join a project demo night & onboarding session listed on{' '}
+            <a
+              href="https://www.meetup.com/code-for-pdx/"
+              target="_blank"
+              rel="noreferrer"
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              meetup
+            </a>{' '}
+            and the{' '}
+            <a
+              style={{
+                color: '#56B6A1',
+                textDecoration: 'none',
+                fontWeight: 600
+              }}
+            >
+              events calendar
+            </a>
+          </li>
+        </ol>
+      </>
+    )
+  },
+  {
+    label: 'Connect on Discord & Request Access',
+    description: (
+      <>
+        <Typography variant="h6">CODE PDX uses Discord for communication</Typography>
+        <ol>
```

Comment:
The console is upset about the standard HTML (again as you're aware). It's saying:
```
<h6> cannot appear as a descendant of <p>
```
and 
```
<ol> cannot appear as a descendant of <p>
```